src/main/java/esa/sen2vm/input/datastrip/DataStripManager.java

```diff
@@ -266,12 +404,11 @@ protected void computeSatelliteQList() throws Sen2VMException {
      * Fill satellitePVList using values from SAD XML file
      * @throws Sen2VMException
      */
-    protected void computeSatellitePVList() throws Sen2VMException {
+    protected void computeSatellitePVList(Boolean activateAvailableRefining) throws Sen2VMException {
         try {
             // Init of used frames
             Frame eme2000 = FramesFactory.getEME2000();
             Frame itrf = FramesFactory.getITRF(IERSConventions.IERS_2010, true);
```

Comment:
mettre le deuxième argument en configuration : simpleEOP = true

Reply:
Fix in commit : https://github.com/sen2vm/sen2vm-core/pull/7/commits/6a2c7d0aac85117b24cc80377c7aaf22aa46ee90